src/encode.rs

```diff
@@ -77,8 +76,11 @@ pub fn perform_encode(keyframes: &[usize], opts: &CliOptions) -> Result<(), Box<
         }
     }
     thread_pool.join();
-    if is_tty(&stderr()) {
-        eprint!("{}", cursor::Goto(1, keyframes.len() as u16 + 5));
+    let term_err = Term::stderr();
+    if term_err.is_term() {
+        term_err
+            .move_cursor_to(1, (keyframes.len() as u16 + 5) as usize)
```

Comment:
It looks like in the console crate, the cursor indices are 0-indexed, while in termion they were 1-indexed. Would you mind correcting these?